src/types/checking.ts

```diff
@@ -0,0 +1,20 @@
+import { WEEK_DAYS_SHORT, WEEK_DAYS_LONG, MONTHS_SHORT, MONTHS_LONG } from "./constants.js";
+import type { WeekDayNamesLong, WeekDayNamesShort, MonthNamesLong, MonthNamesShort } from "./main.js";
+
+const isWeekDayLong = (value: any): value is WeekDayNamesLong => {
+    return Object.values(WEEK_DAYS_LONG).includes(value);
```

Comment:
You should avoid searches like this as many as possible. If I were you I would create separate maps so you could have instant search